build.gradle

```
@@ -1,7 +1,7 @@
 buildscript {
 	dependencies {
-		classpath 'org.kohsuke:github-api:1.114'
-		classpath 'com.guardsquare:proguard-gradle:' + (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_11) ? '7.2.0-beta2' : '7.1.0')
+		classpath 'org.kohsuke:github-api:1.313'
+		classpath 'com.guardsquare:proguard-gradle:7.3.0-beta2')
```

```suggestion
		classpath 'com.guardsquare:proguard-gradle:7.3.0-beta2'
```
Sorry, I didn't remove extra bracket.